### ISSUE TITLE
CertBot Errors

### ISSUE THREAD

**Issue by progkix** [certbot/certbot#6680]
If you're having trouble using Certbot and aren't sure you've found a bug or
request for a new feature, please first try asking for help at
https://community.letsencrypt.org/. There is a much larger community there of
people familiar with the project who will be able to more quickly answer your
questions.

## My operating system is (include version):
Fedora 28 X64

## I installed Certbot with (certbot-auto, OS package manager, pip, etc):
OS package manager

## I ran this command and it produced this output:
**certbot renew**

Traceback (most recent call last):
  File "/bin/certbot", line 11, in <module>
    load_entry_point('certbot==0.29.1', 'console_scripts', 'certbot')()
  File "/usr/local/lib/python3.6/site-packages/pkg_resources/__init__.py", line 487, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/local/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2728, in load_entry_point
    return ep.load()
  File "/usr/local/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2346, in load
    return self.resolve()
  File "/usr/local/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2352, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/usr/lib/python3.6/site-packages/certbot/main.py", line 20, in <module>
    from certbot import account
  File "/usr/lib/python3.6/site-packages/certbot/account.py", line 18, in <module>
    from acme import messages
  File "/usr/lib/python3.6/site-packages/acme/messages.py", line 8, in <module>
    from acme import challenges
  File "/usr/lib/python3.6/site-packages/acme/challenges.py", line 12, in <module>
    import requests
  File "/usr/local/lib/python3.6/site-packages/requests/__init__.py", line 43, in <module>
    import urllib3
  File "/usr/lib/python3.6/site-packages/urllib3/__init__.py", line 8, in <module>
    from .connectionpool import (
  File "/usr/lib/python3.6/site-packages/urllib3/connectionpool.py", line 11, in <module>
    from .exceptions import (
  File "/usr/lib/python3.6/site-packages/urllib3/exceptions.py", line 2, in <module>
    from .packages.six.moves.http_client import (
ModuleNotFoundError: No module named 'urllib3.packages.six'

**certbot --domain books.sdps-bbs.com --apache**

Traceback (most recent call last):
  File "/usr/bin/certbot", line 11, in <module>
    load_entry_point('certbot==0.29.1', 'console_scripts', 'certbot')()
  File "/usr/local/lib/python3.6/site-packages/pkg_resources/__init__.py", line 487, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/local/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2728, in load_entry_point
    return ep.load()
  File "/usr/local/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2346, in load
    return self.resolve()
  File "/usr/local/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2352, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/usr/lib/python3.6/site-packages/certbot/main.py", line 20, in <module>
    from certbot import account
  File "/usr/lib/python3.6/site-packages/certbot/account.py", line 18, in <module>
    from acme import messages
  File "/usr/lib/python3.6/site-packages/acme/messages.py", line 8, in <module>
    from acme import challenges
  File "/usr/lib/python3.6/site-packages/acme/challenges.py", line 12, in <module>
    import requests
  File "/usr/local/lib/python3.6/site-packages/requests/__init__.py", line 43, in <module>
    import urllib3
  File "/usr/lib/python3.6/site-packages/urllib3/__init__.py", line 8, in <module>
    from .connectionpool import (
  File "/usr/lib/python3.6/site-packages/urllib3/connectionpool.py", line 11, in <module>
    from .exceptions import (
  File "/usr/lib/python3.6/site-packages/urllib3/exceptions.py", line 2, in <module>
    from .packages.six.moves.http_client import (
ModuleNotFoundError: No module named 'urllib3.packages.six'

**Comment by progkix** [comment]
I should add that just a few days ago certbot --domain ran without a problem and certbot ran successfully at 0600 this morning.

**Comment by progkix** [comment]
After reviewing all the available messages and browsing source source, I decided to try something radical.

I ran pip uninstall six, pip uninstall urllib3, pip3 uninstall six and pip3 uninstall urllib3.

I followed those up with pip3 install urllib3 and pip3 install six.

This corrected the problem with certbot and things are again running properly. However I found that the uninstall affected another application - Conky. I simply ran pip install urllib3 and pip install six and Conky was once again running.

I don't know why the order suddenly became an important item but in my case I need to have urllib3 and six installed for Python 3 first and Python 2 second.
